test/terramate_google.tm.hcl

```
@@ -0,0 +1,78 @@
+globals {
+  minimum_provider_version    = "3.50"
+  provider_version_constraint = "~> ${global.minimum_provider_version}"
```

```suggestion
  provider_version_constraint = "~> 4.0"
```
This changes ensures using 3.50 for the minimal test and 4.0 for the others.